x/tx/signing/std/handler_map.go

```diff
@@ -2,14 +2,20 @@ package std
 
 import (
 	"cosmossdk.io/x/tx/signing"
+	"cosmossdk.io/x/tx/signing/aminojson"
 	"cosmossdk.io/x/tx/signing/direct"
+	"cosmossdk.io/x/tx/signing/direct_aux"
 	"cosmossdk.io/x/tx/signing/textual"
 )
 
 // SignModeOptions are options for configuring the standard sign mode handler map.
 type SignModeOptions struct {
 	// Textual are options for SIGN_MODE_TEXTUAL
 	Textual textual.SignModeOptions
+	// DirectAux are options for SIGN_MODE_DIRECT_AUX
+	DirectAux direct_aux.SignModeHandlerOptions
+	// AminoJSON are options for SIGN_MODE_LEGACY_AMINO_JSON
+	AminoJSON aminojson.SignModeHandlerOptions
```

Comment:
Can we refactor these options to not duplicate shared options? For instance I imagine that a lot of these will require type resolvers and file resolvers. Let's just have individual fields for each type of option here. Okay to do in a separate PR if that's easier